src/common/com/intellij/plugins/haxe/lang/parser/HaxeGeneratedParserUtilBase.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2017-2017 Ilya Malanin
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intellij.plugins.haxe.lang.parser;
+
+import com.intellij.lang.PsiBuilder;
+import com.intellij.lang.parser.GeneratedParserUtilBase;
+import com.intellij.plugins.haxe.lang.lexer.HaxeTokenTypes;
+
+public class HaxeGeneratedParserUtilBase extends GeneratedParserUtilBase {
```

Comment:
Yeah, I've realized that, during additional testing. I'm already fixed it, but not pushed, because I've worked on solution for #624 (and found it) :-)

You implementation not goint to work too, whitespace will be skipped anyway :-)